### ISSUE TITLE
Issue with latest version of Nagios & statusengine

### ISSUE THREAD

**Issue by jhuber99** [statusengine/broker#33]
Getting the following error trying to use the broker lib with Nagios:
Error: Module '/opt/statusengine/lib/libstatusengine.so' is using an old or unspecified version of the event broker API

broker: 4.0.2
naemon-core: 1.2.3
nagios-plugins: 2.4.3
nagios: 4.4.10

OS: Linux Mint  20.3 (Ubuntu 20.04)

NOTE: the "statusengine" web page comes up, but of course is empty with not broker..

**Comment by nook24** [comment]
Unfortunately we have decided to drop support for Nagios with the next release of the Statusengine broker.
I would recommend to switch to the [Naemon Core](http://www.naemon.io/). It is 1:1 compatible with your Nagios configuration files.

**Comment by sistemmsn** [comment]
Hello, how about is strange? I have it running with nagios core 4.4.10, it works wonderfully, I just had to modify some tables, since it did not allow inserting the data by length, apart from that it does its job

![image](https://user-images.githubusercontent.com/38023703/224609811-0be5b8ef-9e8e-4bbf-804f-15a077ca18d0.png)

Oh unless in their next version of the broker they still leave nagios core aside

**Comment by nook24** [comment]
> I have it running with nagios core 4.4.10, it works wonderfully,

Yes the current version works of Statusengine with Nagios Core. However, this was (and still is) a lot of work to accomplish. I personally have no use of supporting Nagios Core because I haven't used Nagios Core for many years.

The time and motivation of the developers are very limited for this reason we will drop support for Nagios Core. But i would be more than happy if you or someone else will send over pull requests to keep the Nagios Support alive in future versions.

**Comment by marcom29** [comment]
Personally I also use Nagios Core, I don't like Naemon since they use a lot of Patch over Patch and that consumes more resources than necessary. It would be great if they continue with NagiosCore. I know a Nagios Core community that has implemented StatusEngine with Nagios and they love it.

**Comment by sistemmsn** [comment]
> > I have it running with nagios core 4.4.10, it works wonderfully,
> 
> Yes the current version works of Statusengine with Nagios Core. However, this was (and still is) a lot of work to accomplish. I personally have no use of supporting Nagios Core because I haven't used Nagios Core for many years.
> 
> The time and motivation of the developers are very limited for this reason we will drop support for Nagios Core. But i would be more than happy if you or someone else will send over pull requests to keep the Nagios Support alive in future versions.

There is a community of 3277 people who use nagios core, I will try to make your project look more hehehe, I really don't see a joke in neamon, why final is a derivative of nagios as well as, all monitoring programs, icinga, omd ( naemon), Check-mk, zabbix, prtg and op5, etc...

If it could be kept minimal, the worker and broker is more than enough, thanks to those modules we can migrate to the database.

ps since, I will continue with nagios core until I am like the locked cat (with no way out).

Thank you so much

**Comment by sistemmsn** [comment]
> > I don't like Naemon since they use a lot of Patch over Patch and that consumes more resources than necessary
> 
> What do you mean by this? Do you have any example?
> 
> > I know a Nagios Core community that has implemented StatusEngine with Nagios and they love it.
> 
> Would be perfect if they would contact me, to get more help with the Statusengine project :) For now there is no ETA when support for Nagios Core get's dropped, but due to I have no use for it but it makes my life harder, this is inevitable I guess.

We can gladly do something with the project, but I'm not really good at programming, but to get things out of hand, hahaha I can help you test, eh, even with the documentation process so that others can implement the system

**Comment by sistemmsn** [comment]
> > > I don't like Naemon since they use a lot of Patch over Patch and that consumes more resources than necessary
> > 
> > 
> > What do you mean by this? Do you have any example?
> > > I know a Nagios Core community that has implemented StatusEngine with Nagios and they love it.
> > 
> > 
> > Would be perfect if they would contact me, to get more help with the Statusengine project :) For now there is no ETA when support for Nagios Core get's dropped, but due to I have no use for it but it makes my life harder, this is inevitable I guess.
> 
> We can gladly do something with the project, but I'm not really good at programming, but to get things out of hand, hahaha I can help you test, eh, even with the documentation process so that others can implement the system


How can we support you so that you continue to maintain the integration with nagios, we are from Latin America, Spanish-speaking is a great community, as determined by the downloads related to implementing it in nagios

**Comment by gamantilla0475** [comment]
Pertenezco a la comunidad latinoamericana que usa nagios core y esperamos apoyaste en lo que requieras

**Comment by eanselmi** [comment]
> Unfortunately we have decided to drop support for Nagios with the next release of the Statusengine broker. I would recommend to switch to the [Naemon Core](http://www.naemon.io/). It is 1:1 compatible with your Nagios configuration files.

Please, I beg you to reconsider continuing to support Nagios Core. I'm part of a +3000 Nagios Core community with members all over the American continent (we are all Spanish speakers), and your job is crucial to us (we use the status engine for BIG customization and nagios core integrations). Thanks in advance!!!

**Comment by nook24** [comment]
> and your job is crucial to us (we use the status engine for BIG customization and nagios core integrations)

Honestly, this sounds like you are a company to me. I mean, this GitHub repo has 5 stars. 

> I'm part of a +3000 Nagios Core community with members

Sounds great, shouldn't be too hard to find a C developer inside your community :)

**Comment by nook24** [comment]
I'm closing this as Nagios Support has no priority for me. As far as i know Statusengine is working with the current Nagios Version 4.5.14 on Ubuntu 26.04.

https://statusengine.org/tutorials/install-nagios/
